org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -407,6 +464,26 @@ package lang {
 		override method asSet() = self
 
 		override method anyOne() native
+
+		/**
+		 * Returns a new Set with the elements of both self and another collection.
+		 * @returns a Set
+		 */
+		 method union(another) = self + another
```

Comment:
Yes, it's a synonym.
So far Set doesn't have built-in subclasses, so the answer is that it's not overriden yet. It might be in the future.